### PR TITLE
[SMP-1016]: Refactoring Looker Charts

### PR DESCRIPTION
**Pull request by arya-harness**
Included following changes:
- support for mySQL when looker is HA(currently, looker is using single pod pvc support to store data)
- Moved VirtualService and support for istio and secrets from central repos to Dashboards charts so they have ownership of it.
-  ImagePullSecrets support for mySQL as well
- Integrated with harness-commons 1.6.0 so helm upgrades can not fail anymore due to secret not being present

### CODE REVIEW

**Comment by kapilgarg1996** [comment]
Can you update the PR title and add some description about the changes ?

**Comment by arya-harness** [comment]
@kapilgarg1996 done

**Review by sean-harness — APPROVED** [review]

File: src/looker/templates/secrets.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  name: harness-looker-secrets
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "looker.labels" . | nindent 4 }}
+type: Opaque
+data:
+{{- include "looker.generateLookerSecrets2" . }}
+---
+apiVersion: v1
+kind: Secret
```
Comment (kapilgarg1996) [inline src/looker/templates/secrets.yaml:24]: move to mysql folder

**Review by kapilgarg1996 — COMMENTED** [review]
remove charts folder and put them in .gitignore

File: src/looker/templates/secrets.yaml (hunk 1 above)
Comment (arya-harness) [reply src/looker/templates/secrets.yaml]: done

**Review by kapilgarg1996 — APPROVED** [review]

**Pull request merged**